### ISSUE TITLE
A11y_Azure Tools for VS Code_Azure Function_Create New Project_Screen Reader: Voice over is not announcing the dialog names while creating "New project", "Functions" and on "select subscription".

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode-azurefunctions#3819]
### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #AILimited; #Azure Tools for VS Code; #BM-AzureToolforVSCode-MAC-AUG2023; #DesktopApp;  #A11yWCAG1.3.1; #Info and Relationships; #Element:Dailog; #Screen reader; #Voiceover; #AINotInScope; #Device; #MAC;

### OS Details:
Mac OS Ventura 13.5

### Environment Details:
Application Name: Azure Tools for VS Code

### Repro Steps:
1. Turn on Voiceover.
2.Open Visual studio code
3.Click on extension button and search for "Azure Functions"
4. Click on install and move to the "Azure" button
5. Click on "Create New Project Button" from the toolbar.
6.Verify Screen reader is reading the dialog names while creating "New project", "Functions" and on "Subscription".

### Actual Result:
Voice over is not announcing the dialog names while creating "New project", "Functions" and on "Select subscription" like" Create new project", "Create new time Trigger(5/7)" etc.
**Note:** Similar issue is observed in "Create function" and " Select subscription" scenarios also.

### Expected Result:
Voiceover should announce the dialog names while creating "New project", "Functions" and on "Subscription" like" Create new project", "Create new time Trigger(5/7)" etc.

### Attachment:
![A11y_dialog-name](https://github.com/microsoft/vscode-azurefunctions/assets/98530069/d36edf7e-4c9f-4e3a-a147-b94a17be1b6a)

**Comment by MicroFish91** [comment]
Can you please confirm if this is still reproducing?

I have not been able to repro this on macOS v14.0.  When I start from the workspace toolbar, all of the selected commands are read, and once executing the command, all quick pick options are ready in list format.  This also has worked for me when entering through the command palette as well.

**Comment by kupatkar99** [comment]
@MicroFish91 Here i am referring "dialog names" means " Create New project". Please refer highlighted part in below attached screenshot. similarly while creating "Functions" and on "Subscription" like" Create new project", "Create new time Trigger(5/7)" etc.
![Screenshot 2023-10-23 122111](https://github.com/microsoft/vscode-azurefunctions/assets/98530069/779e79c7-281c-4230-8e5a-b27247273664)

**Comment by MicroFish91** [comment]
Similar issue already logged upstream with VS Code, tracking here: https://github.com/microsoft/vscode/issues/149387

**Comment by squ33z3r** [comment]
Add Tag #A11yCAIOH

**Comment by kupatkar99** [comment]
#Closed;
Verified the issue in environment:
Visual studio Code Version: 1.82.1 (Universal)
Issue is fixed. hence closing the bug please find added attcahment for reference.
<img width="1440" alt="MicrosoftTeams-image (1)" src="https://github.com/microsoft/vscode-azurefunctions/assets/98530069/de87ac10-be6f-46e8-bc68-c59b956d0dec">
